package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "jest",
     "eject": "react-scripts eject"
   },
```

Comment:
- [ **OPTIONAL** ]  Good job fixing all changes mentioned by the previous reviewers, However, I think it would be better if you update your `Getting Started` section by adding all information needed by a user such as of how to `run build for production`, `run/use` your app., `run tests`,... I recommend you take a look on this **[template](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)** to have an idea on how to proceed

Reply:
@tresorsawasawa. Thank you so much for taking the time to review my code. Your feedback was incredibly helpful and appreciate your expertise and attention to detail.